internal/file/finder_test.go

```diff
@@ -119,6 +120,48 @@ func TestGetGroups(t *testing.T) {
 	}
 }
 
+func TestGetGroupsPIP(t *testing.T) {
+	setUp(true)
+	path := "testdata/pip"
+	const nbrOfGroups = 2
+
+	lockfileOnly := false
+	fileGroups, err := finder.GetGroups(path, []string{}, lockfileOnly, StrictAll)
+
+	assert.NoError(t, err)
+	assert.Equalf(t, nbrOfGroups, fileGroups.Size(), "failed to assert that %d groups were created. %d was found", nbrOfGroups, fileGroups.Size())
+
+	locksFound := make([]string, 0)
+	manifestsFound := make([]string, 0)
+	for _, fileGroup := range fileGroups.ToSlice() {
+		lockFiles := fileGroup.LockFiles
+		locksFound = append(locksFound, lockFiles...)
+		manifestFile := fileGroup.ManifestFile
+		manifestsFound = append(manifestsFound, manifestFile)
+	}
+	manifestsExpected := []string{"testdata/pip/requirements-dev.txt", "testdata/pip/requirements.txt"}
+	locksExpected := []string{"testdata/pip/requirements-dev.txt.pip.debricked.lock", "testdata/pip/requirements.txt.pip.debricked.lock"}
+	sort.Strings(manifestsExpected)
+	sort.Strings(locksExpected)
+	sort.Strings(manifestsFound)
+	sort.Strings(locksFound)
+	t.Logf("manifest files expected: %s", manifestsExpected)
+	t.Logf("manifest files found: %s, len: %d", manifestsFound, len(manifestsFound))
+	t.Logf("lock files expected: %s", manifestsExpected)
+	t.Logf("lock files found: %s, len: %d", locksFound, len(locksFound))
+	for i := range manifestsExpected {
+		found := filepath.ToSlash(filepath.Clean(manifestsFound[i]))
```

Comment:
This file path correction is needed for the test to run successfully on the Windows platform because it uses backslash instead of slash.